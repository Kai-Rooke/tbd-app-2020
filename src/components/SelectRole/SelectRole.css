.select-role-container {
    background-color: #B2DDDF;
    width: fit-content !important;

}
.SelectRole .largebutton{
width: 1000px;
}
.MobileSelectRole .largebutton{
    width: 100px;
}

@media screen and (max-width: 768px) {
    .extralargebutton
    {


                width: 100%;
    }

  }