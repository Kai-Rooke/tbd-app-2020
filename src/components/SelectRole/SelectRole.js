import React from 'react';
import { Container, Row , Col} from 'react-bootstrap';
import UCPButton from '../Buttons/UCPButton';
import './SelectRole.css'
import ApplicationComplete from '../ApplicationComplete/ApplicationComplete'
//Must Set HREF attribute to use

const SelectRole = () => {
    return(
        <Container className="mt-5 px-3 py-3 select-role-container">
            
            <Row className="text-center">
                <Col className="mt-3 mb-3">
                    <UCPButton to="/login" buttonText="Staff Portal" className="extralargebutton" href="/login" />
                </Col>
                 <Col className="mt-3 mb-3">
                    <UCPButton to="/NewApplicationPage" href="#" buttonText="Prospective Student" className="extralargebutton" props="student" />
                </Col>
            </Row>
        </Container>
    );
}

export default SelectRole;