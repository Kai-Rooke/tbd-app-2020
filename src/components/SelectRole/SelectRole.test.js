import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';
import SelectRole from './SelectRole';

//cleans up 
afterEach(cleanup);

it("Render SelectRole", () => {
    const { asFragment } = render(<SelectRole  />);
    expect(asFragment()).toMatchSnapshot();  
});