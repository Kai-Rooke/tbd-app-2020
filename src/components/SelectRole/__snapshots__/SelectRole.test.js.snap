// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render SelectRole 1`] = `
<DocumentFragment>
  <div
    class="mt-5 px-3 py-3 select-role-container container"
  >
    <div
      class="text-center row"
    >
      <div
        class="mt-3 mb-3 col"
      >
        <a
          href="/login"
        >
          <button
            class="extralargebutton"
            data-testid="ucpButton"
            href="/login"
            type="button"
          >
            Staff Portal​
          </button>
        </a>
      </div>
      <div
        class="mt-3 mb-3 col"
      >
        <a
          href="/NewApplicationPage"
        >
          <button
            class="extralargebutton"
            data-testid="ucpButton"
            href="#"
            props="student"
            type="button"
          >
            Prospective Student​
          </button>
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
